src/features/editUserProfile/ui/form.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+import { useStore } from 'effector-react';
+import { Box, TextField, Container, Typography, Button, EditIcon } from 'shared';
+import { viewerModel } from 'entities/viewer';
+import { User } from 'entities/viewer/model/types';
+import { editUserProfileModel } from 'features/editUserProfile';
+import { DeleteUser } from 'features/editUserProfile/ui/deleteUserButton';
+
+export const EditUserProfile: React.FunctionComponent = () => {
+  const user = useStore(viewerModel.$user);
+  const navigate = useNavigate();
+
+  const editUserProfileFilds: { lable: string; key: keyof User }[] = [
```

Comment:
I guess you can remove "edit" prefix here

Reply:
fixed